week-8/Homework/mandatory/2-fetch-exercise/exercise.js

```diff
@@ -17,10 +17,23 @@ Open index.html in your browser. Every time you refresh the page,
 a different greeting should be displayed in the box.
 */
 
-fetch('*** Write the API address here ***')
-    .then(function(response) {
-        return response.text();
+fetch('https://codeyourfuture.herokuapp.com/api/greetings')
+    .then((response )=> {
+     return response.text();
     })
-    .then(function(greeting) {
-        // Write the code to display the greeting text here
-    });
\ No newline at end of file
+    
+    .then((greeting)=> {
+        
+        addGreeting(greeting)
+    })
+
+    // .catch((error) => {
+    //     console.log(error);
+        
+    // })
+
+    const addGreeting = (greetingObj) => {
```

Comment:
I would call the parameter `greeting` as it is technically a string, not an object.